netbsd/Platform.h

```diff
@@ -94,6 +94,8 @@ static inline void Platform_gettime_monotonic(uint64_t* msec) {
    Generic_gettime_monotonic(msec);
 }
 
+static inline void Platform_dynamicColumnWriteField(ATTR_UNUSED const Process* proc, ATTR_UNUSED RichString* str, ATTR_UNUSED int param) { }
+
 static inline Hashtable* Platform_dynamicMeters(void) {
    return NULL;
 }
```

Comment:
Make things consistent with other platforms …

```suggestion
static inline Hashtable* Platform_dynamicMeters(void) { return NULL; }

static inline void Platform_dynamicColumnWriteField(ATTR_UNUSED const Process* proc, ATTR_UNUSED RichString* str, ATTR_UNUSED int param) { }
```